examples/complete/main.tf

```diff
@@ -1,7 +1,11 @@
+provider "aws" {
+  region = var.region
+}
+
 module "example" {
   source = "../.."
 
-  example = var.example
-
-  context = module.this.context
+  context           = module.this.context
```

Comment:
for consistency (we try to enforce consistency in all modules), move `context` to the last line of the module invocation, similar to https://github.com/cloudposse/terraform-aws-datadog-integration/blob/master/examples/complete/main.tf#L11

Reply:
5aa2f20b0317e4f6c2803e66c6f07e8279a92b25